core/templates/dev/head/domain/utilities/BrowserCheckerService.js

```diff
@@ -40,9 +49,31 @@ oppia.factory('BrowserCheckerService', function() {
     }
   };
 
+  var _supportsSpeechSynthesisInAudioLanguages = function(audioLanguageList) {
```

Comment:
Nope, this service shouldn't know anything about the rest of the app (including the structure of audio language codes and so on). Just hardcode an en-GB check for now. Add a comment above it saying "This should be kept in sync with AUTOGENERATED_AUDIO_LANGUAGES."

Reply:
Done!